IrvinRamiA/Integrator-Project
# ADC Initialization

Closes #2

Definition of done:

- [x] Create the project folder.
- [x] Configuration of ADC.

QA:
- [x] Behave as expected.

Connect a potentiometer with the next configuration:
Pin 1 - 3.3V.
Pin 2 - ADC Channel 0.
Pin 3 - GND.
In the debugger of e2studio add the variables adcData and percentage in real time.
Change the position of the potentiometer and verify that adcData changes from 0 to 255, meanwhile percentage changes from 0 to 100.
